packages/editor/src/ui-tools/tool-bar/tool-bar.ts

```diff
@@ -233,9 +233,8 @@ export class ToolBar extends AbstractUIExtension implements IActionHandler, Edit
   }
 
   editModeChanged(_oldValue: string, _newValue: string): void {
-    if (_oldValue) {
-      this.actionDispatcher.dispatch(SetUIExtensionVisibilityAction.create({ extensionId: ToolBar.ID, visible: true }));
-    }
+    this.containerElement.innerHTML = '';
```

Comment:
fix that the faster toolbar activation will change (remove edit buttons) if the the process editor is in read-only mode

Reply:
-> change moved to https://github.com/axonivy/process-editor-client/pull/445/